vfs/fs_utils.cc

```diff
@@ -537,6 +537,33 @@ dirName(const std::string & filename)
 }
 
 /****************************************************************************/
+/* UX FUNCTIONS                                                             */
+/****************************************************************************/
+
+void
+checkWritability(const std::string & url, const std::string & parameterName)
+{
+    // try to create output folder and write open a writer to make sure 
+    // we have permissions before
+    try {
+        Datacratic::makeUriDirectory(url);
```

Comment:
This will create the directory and the file.  I believe we don't want that.  We can use stat() instead (e.g. via boost::file_system) to determine if the folder has the proper access.  Also, what would be the effect of that call if the path point to S3?